draft-ietf-rats-corim.md

```diff
@@ -1491,36 +1491,97 @@ and how the verifier matches a Reference Value against the Accepted Claims Set
+`measurement-value-map` field within the Conditional Endorsement Triple.
+
+For each Conditional Endorsement Triple, the verifier SHALL attempt to match the
+Reference Value against entries in the Accepted Claims Set (see {{sec-match-one-ref-val}}).
+If it matches then the verifier SHALL add the Endorsed value to the
+Accepted Claims Set (see {{sec-add-to-acs}}).
+
+>> *TODO: Is a Conditional Endorsement Triple affected by `reference-triple-record`s
+in the same group? I think not. Does this need to be mentioned?*
+
+#### Matching a Conditional Endorsement Series Triple against the Accepted Claims Set
+
+A Conditional Endorsement Series Triple {{sec-comid-triple-cond-series}} is an efficient
+encoding of a series of alternative Reference Values, each with a corresponding
+Endorsed Value. The Verifier adds the Endorsed Values corresponding to the first
+Reference Value which matches.
+
+All Reference Values and Endorsed Values in the triple use the same `environment-map`
+which is encoded within the `stateful-environment-record` part of the triple.
+
+Reference Values are encoded in two parts of this triple.
+
+Reference Values which apply to every Endorsed Value are encoded in the
+`stateful-environment-record`, in the same way as for the
+Conditional Endorsement Triple. If any of these Reference Values
+do not match Evidence or Endorsements in the Accepted Claims Set then then the
+entire triple is silently ignored.
+
+Reference Values which apply to only one Endorsed Value are encoded, with their
+matching Endorsed Value, in an array of `conditional-series-record`s.
+Each `conditional-series-record` encodes the `measurement-value-map` part for its
+Reference Values and for the corresponding Endorsed Value.
+
+The entries in the `conditional-series-record` array are processed in order. The verifier
+SHALL attempt to match the Reference Value part of each entry against entries
+in the Accepted Claims Set (see {{sec-match-one-ref-val}}).
+If the Reference Value matches then the verifier SHALL add the corresponding
+Endorsed Value to the Accepted Claims Set (see {{sec-add-to-acs}}) and
+SHALL skip processing the remaining entries in the array.
 
-Each group is processed independently of other groups. If a group fails to match
-the Accepted Claims Set then this does not affect the processing of other groups.
+>> *TODO: Is a Conditional Endorsement Series Triple affected by
+`reference-triple-record`s in the same group? I think not.
+Does this need to be mentioned?*
 
 #### Matching a Reference Value against the Accepted Claims Set {#sec-match-one-ref-val}
 
-This section describes how a Reference Value is matched against Evidence in the Accepted
-Claims Set.
-If any part of the processing indicates that the Reference Value does not match then the remaining steps in this section are skipped for that group.
+This section describes how a Reference Value is matched against Evidence or Endorsements
```

Comment:
What are some use cases for Ref Values matching Endorsements?

Reply:
The main use case is hierarchical CoRIMs. 
- CoMID A has reference values which are expected to match against measurements A and B, if they match it adds endorsement E. 
- CoMID C contains reference values which are expected to match against measurements C and D, and another which is expected to match against endorsement E if authorized by A. If all match then it adds endorsement F.
- The relying party is looking for endorsement F.
I'll put this in a separate PR